RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -68,28 +78,32 @@
 from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
 run2_miniAOD_94XFall17.toModify(
     reducedEgamma, 
-    photonPFClusterIsoSources = cms.VInputTag(
+    photonFloatValueMapSources = cms.VInputTag(
         cms.InputTag("photonEcalPFClusterIsolationProducer"),
         cms.InputTag("photonHcalPFClusterIsolationProducer"),
         ),
-    photonPFClusterIsoOutput = cms.vstring(
+    photonFloatValueMapOutput = cms.vstring(
         "phoEcalPFClusIso",
         "phoHcalPFClusIso",
         ),
-    ootPhotonPFClusterIsoSources = cms.VInputTag(
+    ootPhotonFloatValueMapSources = cms.VInputTag(
         cms.InputTag("ootPhotonEcalPFClusterIsolationProducer"),
         cms.InputTag("ootPhotonHcalPFClusterIsolationProducer"),
         ),
-    ootPhotonPFClusterIsoOutput = cms.vstring(
+    ootPhotonFloatValueMapOutput = cms.vstring(
         "ootPhoEcalPFClusIso",
         "ootPhoHcalPFClusIso",
         ),
-    gsfElectronPFClusterIsoSources = cms.VInputTag(
+    gsfElectronFloatValueMapSources = cms.VInputTag(
         cms.InputTag("electronEcalPFClusterIsolationProducer"),
         cms.InputTag("electronHcalPFClusterIsolationProducer"),
         ),
-    gsfElectronPFClusterIsoOutput = cms.vstring(
+    gsfElectronFloatValueMapOutput = cms.vstring(
         "eleEcalPFClusIso",
         "eleHcalPFClusIso",
         )
     )
+
+from RecoEgamma.EgammaPhotonProducers.reducedEgamma_tools import calibrateReducedEgamma
+from Configuration.Eras.Modifier_run2_miniAOD_94XFall17_cff import run2_miniAOD_94XFall17
+modifyReducedEGammaRun2MiniAOD9XFall17_ = run2_miniAOD_94XFall17.makeProcessModifier(calibrateReducedEgamma)
```

Comment:
I had in mind that all module modifications are moved in here explicitly with .toModify.
The makeProcessModifier is too indirect.
We'll sort out later. OK for now.

Reply:
okay thanks for letting it through.

 I thought though this was what they said to do in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCmsDriverEras

At the very least here at least folks know that a modifier exists